src/Pester.Runtime.ps1

```diff
@@ -170,15 +170,18 @@ function New-ParametrizedBlock {
         [String[]] $Tag = @(),
         [HashTable] $FrameworkData = @{ },
         [Switch] $Focus,
-        [String] $Id,
         [Switch] $Skip,
         $Data
     )
 
+    # using the position of Describe/Context as Id to group data-generated blocks. Should be unique enough because it only needs to be unique for the current block, so the way to break this would be to inline multiple blocks with ForEach, but that is unlikely to happen. When it happens just use StartLine:StartPosition
+    # TODO: Id is used by NUnit2.5 and 3 testresults to group. A better way to solve this?
+    $id = $StartLine
+
```

Comment:
Should we just rename the Id-property to `GroupId` and use a guid to make it more obvious? It's a public property, but I doubt anyone relies on it and the fact that it's startline atm.